Dockerfile

```diff
@@ -16,6 +16,7 @@ ENV BUILD_NUMBER ${BUILD_NUMBER:-1_0_0}
 RUN apt-get update && \
     apt-get -y upgrade && \
     apt-get install -y curl && \
+    apt-get install -y libreoffice && \
```

Comment:
Not sure about this - generally a docker container would have one purpose (in this case running our Koltin application) so maybe libreoffice should be in its own container?

Reply:
@JamesBoobier  Not sure can we have this separate container, we can connect after the standup to discuss on this!

Reply:
As discussed we'll think about this later